.github/workflows/pr.yml

```diff
@@ -0,0 +1,27 @@
+name: Test PR
+
+on: [pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: install pytest
+      run: pip install pytest
+    - name: install package
+      run: pip install -r requirements.txt .
+    - uses: ricardochaves/python-lint@v1.4.0
+      with:
+        python-root-list: "dfpl tests"
```

Comment:
Another thing we need to do: See if all tests are up to date and pass. I can already see that they won't.

Reply:
Sure. Do you have a command line for me that starts the tests?

Reply:
It seems you possess the right street-smarts to have found this out already :)